examples/penguins/app.py

```diff
@@ -12,6 +14,13 @@
 import shiny.experimental as x
 from shiny import App, Inputs, Outputs, Session, reactive, render, req, ui
 
+# There is a matplotlib bug which causes pyright failures
```

Comment:
Small nit: the warning actually has nothing todo with pyright - it just happens to be on a line where there is a pyright comment.

Reply:
Oh right, thanks for the clarification